pro/src/ui-kit/form/SelectAutocomplete/SelectAutocomplete.tsx

```diff
@@ -71,11 +72,20 @@ const SelectAutocomplete = ({
       onSearchChange()
       return
     }
-    const regExp = new RegExp(searchField.value, 'i')
+
+    const formattedValue: string = toLowerCaseWithoutAccents(searchField.value)
+
+    const formattedValues = formattedValue.split(' ')
+
     setFilteredOptions(
-      options.filter(
-        option => searchField.value === '' || option.label.match(regExp)
-      )
+      options.filter(option => {
+        const formattedLabel = toLowerCaseWithoutAccents(option.label)
+
+        return (
+          formattedValues.length === 0 ||
+          formattedValues.every(word => formattedLabel.includes(word))
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** Est ce qu'on veut bien `every` ici ? Ou bien on veut être plus permissif avec `some` ?
Ou peut être une combinaison des deux (facile à dire haha) où on met en premier les résultats qui matchent `every` puis ceux qui matchent `some`‏

Reply:
Yes on veut bien que chaque mot soit contenu dans le libellé :) du moins pour l'instant